Zadanie7.3/businessCard.py

```diff
@@ -0,0 +1,30 @@
+#================================================================
+class BaseContact:
+    def __init__(self, first_name, last_name, phone_number, email):
```

Comment:
proszę dodać typy argumentów i typy zwracanych przez metody wartości we wszystkich plikach